src/services/JFrogModalService.js

```diff
@@ -1,42 +1,8 @@
-import JFrogConfirmModalComponent from '../components/JfConfirmModalComponent/index.vue'
-import JFrogCodeModalComponent from '../components/JfCodeModalComponent/index.vue'
-import JfFullTextModalComponent from '../components/JfFullTextModalComponent/index.vue'
-import JfDynamicModalComponent from '../components/JfDynamicModalComponent/index.js'
-import JfWizardModalComponent from '../components/JfWizardModalComponent/index.vue'
+import JFrogModalFactory from "./JFrogModalFactory.js"
```

Comment:
Not very content with this solution, as we spoke before.
This is not solving the testability problem as JFrogModalFactory.js wont be testable because of the same problem that created  JFrogModalFactory.js in the first place.
We had a half hour discussion about this problem with @dannyreiser and @tomerelkayam and we see that the right solution is to pass inner modal template component as prop or with slot.
From practical perspective, maybe this change can live by now as we don't have the time to right it properly.
Your decision...

Reply:
merging with this change for now. Will take up the comment offline and change if necessary.